### ISSUE TITLE
issues with classes from non java source: add option to exclude class entirely?

### ISSUE THREAD

**Issue by bavardage** [palantir/gradle-revapi#41]
## What happened?

I'm using gradle-revapi in a project which contains some scala code (because it involves some interactions with apache spark catalyst, which is a pretty scala heavy api).

I added a new constructor to a scala class, which takes an extra parameter (but keeping the old constructor too), which resulted in revapi detecting a large number of 'breaks', seemingly because some of the methods scala is creating behind the scenes changed - e.g. I see breaks such as:
```
- code: "java.method.removed"
      old: "method long scala.runtime.AbstractFunction1<T1, R>::apply$mcJJ$sp(long)\
        \ @ com.palantir.MyScalaClass$"
      new: null
```

## What did you want to happen?

Ultimately it might be nice to try to use mima (https://github.com/lightbend/mima) for something like this.

For now it might be nice to be able to just exclude classes entirely from revapi checking (in this case there is no public facing api/abi change). This would avoid filling up `revapi.yml` with a large number of these breaks.

**Comment by CRogers** [comment]
Perhaps an even better solution is perhaps to ignore all Scala classes, as revapi doesn't support/doesn't with Scala?

**Comment by CRogers** [comment]
Fixed in 1.0.4 (for scala)
